api/auth/external_auth.proto

```diff
@@ -125,7 +119,11 @@ message AttributeContext {
   Peer destination = 2;
 
   // Represents a network request, such as an HTTP request.
-  Request request = 4;
+  oneof req {
```

Comment:
Please remove this oneof. HTTP, TCP, TLS are different layers. They are not oneof.